core/trino-main/src/main/java/io/trino/sql/planner/optimizations/SymbolMapper.java

```diff
@@ -371,6 +373,42 @@ public TableWriterNode map(TableWriterNode node, PlanNode source, PlanNodeId new
                 node.getStatisticsAggregationDescriptor().map(descriptor -> descriptor.map(this::map)));
     }
 
+    public MergeNode map(MergeNode node, PlanNode source)
+    {
+        return map(node, source, node.getId());
+    }
+
+    public MergeNode map(MergeNode node, PlanNode source, PlanNodeId newId)
```

Comment:
This method is not used otherwise than by the method above. Please inline it.

Reply:
Inlined.